synapse/models/material.py

```diff
@@ -31,37 +31,88 @@ class MatModule(s_module.CoreModule):
 
     def getModelDefs(self):
         modl = {
+
+            'interfaces': (
+
+                ('phys:object', {
```

Comment:
Do we want a :name and :desc prop?

Reply:
I don't think it's safe yet to assume that every instance of a physical object will need a `:name` and `:desc` field. For example, `transport:air:plane`.